packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetail/AccordianText.tsx

```diff
@@ -86,7 +86,12 @@ export default function AccordianText(props: AccordianTextProps) {
   const styles = useStyles2(getStyles);
   return (
     <div className={className || ''}>
-      <div className={cx(styles.header, headerClassName)} {...headerProps} data-test-id="AccordianText--header">
+      <div
+        className={cx(styles.header, headerClassName)}
+        {...headerProps}
+        data-testid="AccordianText--header"
```

Comment:
same as before, `data-testid` is the correct format so i'd remove the `data-test-id` one and fix anything that needs fixing 🙂